logging/log.go

```diff
@@ -52,7 +52,7 @@ type Level uint32
 // Create a general Base logger
 var (
 	baseLogger      Logger
-	telemetryConfig TelemetryConfig
+	telemetryConfig *TelemetryConfig
```

Comment:
Maybe this one should actually be owned here? Is this the one canonical value of TelemetryConfig, and we should pass references to this value? Is there one part of the code that owns the value if not here?

Reply:
It is not clear when this global is accessed. And when this goes out of sync with a local variable, then it can be a bug. 
I will to do away with this global variable.